subsurface_enabled_translations

```diff
@@ -1,12 +1,138 @@
+qt_ar.qm
+qtbase_ca.qm
+qtbase_cs.qm
+qtbase_de.qm
+qtbase_en.qm
+qtbase_fi.qm
+qtbase_fr.qm
+qtbase_he.qm
+qtbase_hu.qm
+qtbase_it.qm
+qtbase_ja.qm
+qtbase_ko.qm
+qtbase_lv.qm
```

Comment:
This is an interesting question... should we only add those languages that we have our own translations for? Or all the languages that Qt support? This includes Korean and Japanese, for example, which we don't support. I think it would be better to only include those languages for which we have "good enough" (I usually say >70% of the strings translated) translations.